.circleci/config.yml

```diff
@@ -1122,10 +1122,3 @@ workflows:
           report: true
           requires:
             - Build app - Linux (stage)
-      # e2e desktop tests on exe build
-      - e2e-exe:
```

Comment:
I think better comment this test
Someday it will return

Reply:
agree. thanks!